### DISCUSSION TITLE
panda emit-pkg not getting --base from panda.config outdir

### DISCUSSION THREAD

**Discussion by AdamQuadmon** [chakra-ui/panda#2897] [Q&A] [answered]
When I run `panda emit-pkg`, I expect the generated export paths to have as base path the value of `outdir` in `panda.config`, but it uses the root of the package as base.

Is this by design? a bug? or something wrong with my implementation?

I found a `--base` flag which let me add the value manually, but there is no reference in documentation

**Comment by anubra266** [comment, marked as answer]
@AdamQuadmon `--base` is The base directory of the package.json entrypoints, which we asuume to be root by default. I'll update the docs
